client/.storybook/main.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.js'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-postcss'
```

Comment:
I got the following message when running Storybook without this addon. But to be honest, I'm not sure if we need it.

<img width="839" alt="Screen Shot 2021-04-25 at 02 09 35" src="https://user-images.githubusercontent.com/25715018/115971642-d21efb80-a573-11eb-93b8-58689f5aa066.png">

Reply:
Other than the error, we would probably need it for tailwind.

Reply:
Without addon-postcss, the code will throw a deprecation warning. It is needed